bash/bldjar

```diff
@@ -5,26 +5,41 @@
 set -e              # exit on all errors
```

Comment:
I think this statement causes bash to abort the procedure on any error?
It that is so, then would it return to the caller without resetting the current path to its prior value?